google_project/variables.tf

```diff
@@ -78,4 +78,15 @@ variable "project_services" {
   description = "List of google_project_service APIs to enable."
   default     = []
   type        = list(string)
-}
\ No newline at end of file
+}
+
+variable "realm" {
+  description = "Realm is a grouping of environments being one of: global, nonprod, nonprod"
```

Comment:
```suggestion
  description = "Realm is a grouping of environments being one of: global, nonprod, prod"
```

Reply:
what happens if a GCP project being created is not environment specific - that always defaults to global then?

Reply:
In what scenario were we would provision resources that aren't environment specific? I am assuming for something like shared resources? Then if so global should be used.

Reply:
so we don't use a separate project per environment for web SRE atm. is the idea that this needs to switch then?

Reply:
(still approving to move forward, but this is something that's been on my mind reviewing this terraform for cloudops-based mentalities)

Reply:
> so we don't use a separate project per environment for web SRE atm. is the idea that this needs to switch then?

Yes. From a security perspective, in the past (no record of this decision of-course :( ) we wanted segmentation between these environments. From a cost perspective, it made it easier to determine costs between them.